src/lib/y2partitioner/actions/clone_disk.rb

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast/i18n"
+require "y2partitioner/dialogs/disk_clone"
+require "y2partitioner/actions/controllers/disk_device"
+
+Yast.import "Popup"
```

Comment:
JFYI yast2 now have more modern Popup class - https://github.com/yast/yast-yast2/blob/master/library/general/src/lib/yast2/popup.rb

Reply:
Yes, probably we should start to use it everywhere.

Reply:
well, for old code now it is not ideal time, but for new code it makes sense.